wikimedia/wikipedia-ios
# [ALT TEXT] Alt text preview step

**Phabricator:** https://phabricator.wikimedia.org/T370359

### Notes
* This PR adds the review step to the alt text flow
* Publishing alt text now works from this view
* I'm downloading the full image at the modal to prevent downloading two images, even if the modal only would need the thumbnail

### Test Steps
1. Go through the alt text flow
2. Verify the preview matches the designs
3. Verify you can return to the modal sheet and that the alt text is pre-filled in the text field
4. Publish and verify your edit
5. You should go back to the image recs flow, to the next recommendation
